front/src/Components/Header.js

```diff
@@ -6,64 +6,69 @@ import Toolbar from 'material-ui/Toolbar';
 import Typography from 'material-ui/Typography';
 import Grid from 'material-ui/Grid';
 import Button from 'material-ui/Button';
-import { grey } from 'material-ui/colors';
+import { grey, orange } from 'material-ui/colors';
```

Comment:
./src/Components/Header.js
  Line 9:  'orange' is defined but never used  no-unused-vars